snowflake/ingest/simple_ingest_manager.py

```diff
@@ -30,6 +27,13 @@
 from logging import getLogger
 logger = getLogger(__name__)
 
+try:
+    from typing import Text, Dict, Any
+except ImportError:
+    logger.debug('# Python 3.5.0 and 3.5.1 have incompatible typing modules.', exc_info=True)
+    from typing_extensions import Text
+    from typing import Dict, Any
```

Comment:
If Text is the only offending module, why not moving `from typing import Dict, Any` out of the try catch block.

Reply:
Fixed.